### ISSUE TITLE
Should UHTtheme be revised from scratch?

### ISSUE THREAD

**Issue by jeffreybarry** [jeffreybarry/humantrafficking#5]
I'm mildly uncomfortable with the UHTtheme being based on the portfolio theme. That's a good quick start for use during Iliads week but, long term, seems like a fresh base theme would be suitable. 

Perhaps starting with an [Underscores based theme](https://underscores.me/), adding in Bootstrap (especially for the cards functionality), and then styling from there. The custom category pages and and other customized theme files adapted during Iliads week should be easily portable to a new theme.

**Comment by sbufkin** [comment]
I agree. I've started working on an Underscores based theme with Bootstrap. I've got pretty much all the functionality we had in the portfolio theme, as well as some improvements (the dynamic content below entries was way easier). I've also made some progress with some basic styling that we can build on. I'm still putting all the pieces together, but when I've gotten a little further, I'll create a repository for the new theme.

**Comment by sbufkin** [comment]
I've finished preliminary development on the new uht theme. There's a new repo for the theme, as well as a list of initial issues to be resolved. I've also uploaded the theme to the toolbox, so you can see how it functions on the site.
